components/reactCalculator.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import { ADD, SUBTRACT, MULTIPLY, DIVIDE } from '../public/javascript/operatorConstants'
+import calculator from '../public/javascript/calculator'
+
+const validateInputs = ({ num1, num2 }) => ((
+  num1.length > 0 && Number.isNaN(num1)
+) && (num2.length > 0 && Number.isNaN(num2)))
```

Comment:
This is not working as expected

![image](https://user-images.githubusercontent.com/5775716/83045829-2512ca80-a014-11ea-9e71-23ff7317424d.png)